praw/reddit.py

```diff
@@ -694,7 +694,13 @@ def info(
 
             api_parameter_name = "id" if is_using_fullnames else "sr_name"
 
-            def generator(names: Iterable[str | praw.models.Subreddit]):
+            def generator(
+                names: Iterable[str | praw.models.Subreddit],
+            ) -> Generator[
```

Comment:
There was a reason for utilizing `Iterator` over `Generator`. Leaving a comment to remind me to find it. @jarhill0 was the one who suggested using `Iterator` over `Generator`.

Reply:
Types of `Generator[T, None, None]` can be shortened to `Iterator[T]`. I don't really have a preference either way.

Reply:
Ah, I don't have a preference either other than being consistent.

Reply:
I'll shorten them all tomorrow.